src/dedalus.js

```diff
@@ -548,37 +548,51 @@ var Dedalus,
     };
 
     /**
-     * Remove an item from the inventory and trigger a "modified invetory" event
+     * Remove an item from the inventory and trigger a "modified inventory" event
      * on the message center
      * @param  {String} object Id of the object to remove
      */
     Dedalus.prototype.removeFromInventory = function (object) {
         var i;
 
-        for (i = 0; i < this._story.inventory.length; i += 1) {
+        //console.log("Removing " + object + "from inventory. Start: [" + this._story.inventory + "], length: " + this._story.inventory.length);
```

Comment:
Could you please remove all the commented out console.log statements, please? In this project, comments are only used as documentation.
Thanks

Reply:
Sure thing, I've updated the pull. Thank you for your patience!